### ISSUE TITLE
Apple Pay Direct and Phone Number Required setting lead to failed orders

### ISSUE THREAD

**Issue by fjbender** [mollie/Shopware#306]
Steps to reproduce:
* Activate "Phone Number Required" in Shopware Storefront->Sign-In / Sign Up settings
* Activate Apple Pay Direct
* Try to pay with Apple Pay direct

Expected result:
* If a phone number is given in the Apple Pay data, order is successful
* If no phone number is available in the Apple Pay data, order is denied, a meaningful error message is displayed

Actual result:
* No error message is displayed, one gets redirected to /checkout/confirm
* The following exception is logged
```
[2022-01-05T13:18:25.623471+01:00] Mollie.ERROR: Error starting Mollie Apple Pay Direct payment (Session: ar7t...) {“error”:“phone: \n”,“session”:“ar7ti3qjfdfk5q9gs25ehnbcln”,“processors”:{“uid”:{“uid”:“b26ff33"},“web”:{“url”:“/MollieApplePayDirect/startPayment”,“ip”:“127.0.0.*“,”http_method”:“POST”,“server”:“shop.example.org”,“referrer”:“https://shop.example.org/artikel/118/cashew-erdnuss-orange-kakao-cubes?c=78”},“introspection”:{“file”:null,“line”:null,“class”:“Shopware_Controllers_Frontend_MollieApplePayDirect”,“function”:“startPaymentAction”}}} []
```

**Comment by boxblinkracer** [comment]
thanks for this.
...funny what is all possible :D but shouldnt be a problem

**Comment by boxblinkracer** [comment]
happy to announce it's been released in v2.0 :)
